src/filing/institutions/container.jsx

```diff
@@ -4,14 +4,17 @@ import requestInstitutions from '../actions/requestInstitutions.js'
 import fetchEachInstitution from '../actions/fetchEachInstitution.js'
 import receiveInstitutions from '../actions/receiveInstitutions.js'
 import Institutions from './index.jsx'
-import { getKeycloak } from '../utils/keycloak.js'
+import { getKeycloak, logout } from '../utils/keycloak.js'
+import Loading from '../../common/LoadingIcon.jsx'
 
 export class InstitutionContainer extends Component {
   componentDidMount() {
+    if(this.props.maintenanceMsg) return logout()
```

Comment:
I don't think that this should be handled in the institutions container, I'd say it would be more effective in componentDidMount and componentDidUpdate of `./src/filing/App.jsx` (which will also apply to people who are already on the submission page, rather than people just entering the app via the institutions page).